pydmconverter/widgets.py

```diff
@@ -520,19 +516,19 @@ def generate_properties(self) -> List[ET.Element]:
             A list of XML elements representing the PyDMPushButton properties.
         """
         if self.is_off_button is not None and not self.is_off_button:
```

Comment:
```suggestion
        if self.is_off_button is not None:
            show_button = not self.is_off_button
            self.rules.append(RuleArguments("Visible", self.channel, False, show_button, None, None))
            self.rules.append(RuleArguments("Enable", self.channel, False, show_button, None, None))
```

Here's a way to remove some duplicate code by replacing the if-else statement below with a bool.